controllers/nodefeaturediscovery_resources.go

```diff
@@ -58,8 +62,17 @@ func Add3dpartyResourcesToScheme(scheme *runtime.Scheme) error {
 	return nil
 }
 
+// filePathWalkDir takes a path as an input and finds all files
+// in that path, but not directories
 func filePathWalkDir(root string) ([]string, error) {
+
+	// files contains the list of files found in the path
+	// 'root'
```

Comment:
This is quite self-evident, when reading the code and/or function description and I'd suggest just to drop it.